README.md

```diff
@@ -116,6 +134,220 @@ viewModel x = div_ [] [
  , button_ [ onClick SubtractOne ] [ text "-" ]
  ]
 ```
+## Quick start
+To get started quickly building applications, we recommend using the [`stack`](https://docs.haskellstack.org/en/stable/README/) or [`nix`](https://nixos.org/nix) package managers. Obtaining [`GHCJS`](https://github.com/ghcjs/ghcjs) is required as a prerequisite. `stack` and `nix` make this process easy, if you're using `cabal` we assume you have [obtained `GHCJS`](https://github.com/ghcjs/ghcjs#installation) by other means.
+
+All source code depicted below for the quick start app is available [here](https://github.com/dmjio/miso/tree/master/sample-app).
+
+### Stack
+In the `miso` repository there is a [folder named `stack`](https://github.com/dmjio/miso/tree/master/stack) with "known to work" configurations for `GHCJS`. One stack file exists for both the `7.10.3` and `8.0.1` versions of `GHCJS`. In general, we recommend developing with the `7.10.3` version since it currently supports `GHCJSi` (a REPL that connects to the browser by way of a [`nodejs`](https://nodejs.org/en/) web server using [`socket.io`](https://socket.io/)) and building with the `8.0.1` version (if possible). For more information on using `stack` with `GHCJS`, please consult the [GHCJS section of the `stack` docs](https://docs.haskellstack.org/en/stable/ghcjs/).
+
+To begin, create the following directory layout
+```bash
+➜  mkdir app && touch app/{Main.hs,app.cabal,stack.yaml} && tree app
+app
+|-- Main.hs
+|-- app.cabal
+`-- stack.yaml
+```
+
+Add a `stack.yaml` file that uses a recent version of `miso`.
+```bash
+➜  cat app/stack.yaml
+resolver: lts-6.20
+compiler: ghcjs-0.2.0.9006020_ghc-7.10.3
+compiler-check: match-exact
+
+packages:
+ - '.'
+extra-deps: []
+
+setup-info:
+  ghcjs:
+    source:
+      ghcjs-0.2.0.9006020_ghc-7.10.3:
+         url: http://ghcjs.tolysz.org/lts-6.20-9006020.tar.gz
+         sha1: a6cea90cd8121eee3afb201183c6e9bd6bacd94a
+
+- location:
```

Comment:
Is there a reason to choose this specific commit? If not I would suggest to just add an entry in the `extra-deps` section that fetches `miso-0.4.0.0` from hackage.